### ISSUE TITLE
Change volume before container start

### ISSUE THREAD

**Issue by imaia** [docker/compose#4276]
Imagine you have a container with your project and that project has a few static files that you want served by a http server. Then you have another container, a nginx container, that has access to the volume with the static files through volumes_from. Imagine you want to update your volume with content every time you reload your container. What should you do? Right now, for what I see, you have to remove the container and add it again. Is there another way? Shouldn't there be a easier way?

**Comment by shin-** [comment]
Hi @imaia ,

In simpler terms, are you trying to say you would like to change the (filesystem) contents of a running container on the fly? Maybe you could mount a host directory containing the files you want to change periodically?

**Comment by imaia** [comment]
@shin- Kind of. I was copying stuff to the volume during build. As it seems, that kind of doesn't work. So, I just moved my copying to the entrypoint, and all went well. I was trying to use a patched project, where the patch was being applied during de build, which kind of doesn't work (except during container creation).
